src/ir/comp.rs

```diff
@@ -337,6 +349,16 @@ impl Bitfield {
     pub fn name(&self) -> &str {
         self.data.name().unwrap()
     }
+
+    /// Name of the generated Rust getter for this bitfield.
+    pub fn getter_name(&self) -> &str {
+        self.getter_name.as_ref().unwrap()
+    }
+
+    /// Name of the generated Rust setter for this bitfield.
+    pub fn setter_name(&self) -> &str {
+        self.setter_name.as_ref().unwrap()
```

Comment:
Let's replace these `unwrap`s with `expect`s that detail the invariant that is being broken if this panics:

```rust
self.setter_name
    .as_ref()
    .expect("`Bitfield::setter_name` should only be called after assigning field names")
```

And the same for `getter_name`.

Reply:
Yeah, sure!